bot/README.md

```
@@ -1,186 +1,176 @@
-   cd bot
-   ```
+---
 
-2. **Create a virtual environment (recommended):**
+## 🛠️ Installation
 
-   ```bash
-   python -m venv .venv
-   ```
+1.  **Navigate to the bot directory:**
+    ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)